requirements.txt

```diff
@@ -19,3 +19,4 @@ sseclient-py==1.7
 toposort==1.5
 urllib3==1.22
 containerregistry-ccwienk==0.2.1
+bcrypt==3.1.4
```

Comment:
this change ought to be separately pushed to master (no review required), followed by a release of cc-job-image + incrementing the default cc-job-image version (so the runtime dependency is present in all relevant execution environments for cc-utils).

Reply:
done